mappings/7507734918533b3b896241b4704f3d4ce805256b01da6fcede43043642616279534e454b.txt

```diff
@@ -0,0 +1,44 @@
+{
+    "subject": "7507734918533b3b896241b4704f3d4ce805256b01da6fcede43043642616279534e454b",

```

Comment:
Hello @JimBobCorn,

1. Your file has a .txt extension
2. The token you are trying to change already exists with the same subject name:
7507734918533b3b896241b4704f3d4ce805256b01da6fcede43043642616279534e454b.json
If you are trying to update values for that token, please create the file with the .json extension, increment sequenceNumber for the updated values, and sign with the valid key. You can find a detailed guide here: https://github.com/cardano-foundation/cardano-token-registry/wiki/How-do-I-update-my-entry-in-the-registry%3F